frontend/gateway/gateway.go

```diff
@@ -251,18 +251,13 @@ func (gf *gatewayFrontend) Solve(ctx context.Context, llbBridge frontend.Fronten
 		}
 	}
 
-	lbf, ctx, err := serveLLBBridgeForwarder(ctx, llbBridge, gf.workers, inputs, sid, sm)
-	defer lbf.conn.Close() //nolint
+	lbf, ctx, err := serveLLBBridgeForwarder(ctx, llbBridge, exec, gf.workers, inputs, sid, sm)
+	defer lbf.conn.Close() // nolint
```

Comment:
Careful, adding a space between `//` and the comment text semantically changes it from a (linter) directive to a comment for humans. Your editor might have done it automatically [because it doesn't match the directive regex defined in versions of Go newer than the Buildkit v0.10 branch.](https://tip.golang.org/doc/comment#:~:text=Directive%20comments%20such,toolname%3Adirective.)